### ISSUE TITLE
Any idea how to animate box-shadow?

### ISSUE THREAD

**Issue by ryanflorence** [animatedjs/animated#56]
It's a silly rule since it doesn't have box-shadow-radius, spread, color, etc.

```
        boxShadow: anim.interpolate({
          inputRange: [ 0, 1 ],
          outputRange: [
            '0px 0px 0px hsla(0, 0%, 0%, 0.5)',
            '0px 0px 40px hsla(0, 0%, 0%, 0.5)'
          ]
        }),
```

That would be cool! but seems heavy to have to dig in and find every possible number that could be interpolating :\

Any ideas?

**Comment by browniefed** [comment]
@ryanflorence there is a PR open for it

**Comment by browniefed** [comment]
Ah has a conflicting issue, needs a rebase.

**Comment by arcanis** [comment]
If targeting the web, you could be using the `Animated.template` function to support it. Something like this would yield the result in your first post, but you could add other values to interpolate the other properties as well:

```js
let value = anim.interpolate({ inputRange: [ 0, 1 ], outputRange: [ 0, 40 ] });
...
style.boxShadow = Animated.template `0px 0px ${value}px hsla(0, 0, 0, 0.5)`;
```

**Comment by browniefed** [comment]
Nothing that specific is actually needed. Just need to fix the interpolation to understand multi-values.
https://github.com/animatedjs/animated/pull/47

**Comment by arcanis** [comment]
Sure, in this specific case it's only a workaround :)

**Comment by browniefed** [comment]
Yeah good point, I believe Leland Richardson had ideas about all of this. Although mostly in the form of building out new `Animated.Value` abilities w/ `Animated.Shape`

Lots of use cases to be covered, especially SVG paths which I think is a huge one.

**Comment by souporserious** [comment]
I'd love to help out with SVG interpolation if it's still needed :) I've been looking at a lot of libs that support this. I'm guessing it would need to work in both React Native and React Web? I _should_ be able to handle the web side or at least start it, I'd be willing to look into RN as well and try to learn some things.
